app/components/report-row/template.hbs

```diff
@@ -15,6 +15,11 @@
     onchange={{action (mut changeset.comment) value='target.value'}}
     data-test-report-comment
   >
+  {{#if changeset.task.project.customerVisible }}
+    <div class="customer-visible-icon">
```

Comment:
Could you please add a `title` to the `div`? Something like `This project is visible to the customer` to make it more explanatory.
And when you're at it, I would prefer if you could create a component for the implementation of the icon and reuse that everywhere else instead of creating a `div` each time manually. That way the code will be less redundant and more maintainable for the future.

Reply:
done